.travis.yml

```diff
@@ -69,11 +69,10 @@ jobs:
           then
             scripts/build_doxygen.sh ${DOXYGEN_VERSION}
           fi
-        - export PATH="$PATH:${TRAVIS_BUILD_DIR}/doxygen/build/bin"
       script:
         - echo $PATH
-        - doxygen --version
-        - scripts/run_doxygen.sh
+        - ${TRAVIS_BUILD_DIR}/doxygen/build/bin/doxygen --version
+        - scripts/run_doxygen.sh ${TRAVIS_BUILD_DIR}/doxygen/build/bin/doxygen
```

Comment:
As far as I understand (and my bash is pretty rusty), this commit isn't needed. Please explain why you want to make this change, preferably in the commit message.

Reply:
I assume this commit is needed to make sure that the newly built version of doxygen is picked up, rather than a pre-installed version (which would appear sooner on the PATH). If that's the case, then an alternative solution would be to just re-write the `export PATH` as : `export PATH="${TRAVIS_BUILD_DIR}/doxygen/build/bin:${PATH}"`

Reply:
I guess also that the added bonus here of explicitly specifying which doxygen binary is used by the run_doxygen.sh script, is that we make sure that script always works in that way for developers too.

Reply:
Yes it isn't needed - this all works now. The intent was to:
 - make it clearer which version is being used
 - help developer know how they can run a specific version of it using the script